test/functional/readpreference_tests.js

```diff
@@ -546,4 +547,18 @@ describe('ReadPreference', function() {
       });
     }
   });
+
+  it('Should throw an error on an invalid readPreference', function(done) {
+    const configuration = this.configuration;
+
+    const client = configuration.newClient();
+    client.connect((err, client) => {
+      const db = client.db(configuration.db);
+      expect(db.collection.bind(db, 'test', { readPreference: 'invalid' })).to.throw(
```

Comment:
did you mean to call `collection.bind` here?

Reply:
Yes, so I can check that it is throwing the `readPreference` error.